docs/SlavaUkraini/examples/video-overlay/index.md

```diff
@@ -1,19 +1,79 @@
 ---
 layout: tutorial_v2
-title: Leaflet on Mobile
+title: Overlay
 ---
 
-## Video on webpages
+## Overlay - Tutorial
 
-Video used to be a hard task when building a webpage, until the [`<video>` HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) was made available.
+There are 3 Overlays in Leaflet API: 
+- [`ImageOverlay`](/reference.html#imageoverlay): Raster Layer, Extends [`Layer`](/reference.html#layer)
+- [`VideoOverlay`](/reference.html#videooverlay): Raster Layer, Extends [`ImageOverlay`](/reference.html#imageoverlay)
+- [`SVGOverlay`](/reference.html#svgoverlay): Vector Layer, Extends [`ImageOverlay`](/reference.html#imageoverlay)
 
-Nowadays, we can use the following HTML code:
+In this tutorial, you’ll learn how to use these Overlays.
+
+### `ImageOverlay`
```

Comment:
Ideally this section should also have a live example, in a new `example-image.md`.

Reply:
I agree.
- image example added.